packages/runtime/src/next/api-handler/index.ts

```diff
@@ -65,14 +67,14 @@ export function MakeswiftApiHandler(
       )
     }
 
-    const client = new Makeswift(apiKey, { apiOrigin })
+    const client = new Makeswift(apiKey, { apiOrigin, unstable_previewData: req.previewData })
```

Comment:
Just to make sure I am understanding currently:

1. proxy passes `unstable_siteVersion: MakeswiftSiteVersion.Working` as previewData 
2. API handler passes that along to the MakeswiftClient 
3. This add headers to how MakeswiftClient fetches data from our API, which results in preview data instead of production data.

Reply:
Yup!